pom.xml

```diff
@@ -60,16 +60,16 @@
 
         <mockito.version>5.14.1</mockito.version>
 
-        <!-- temporary version waiting for real automatic release in ecocode repository -->
-        <ecocode-rules-specifications.version>1.6.4</ecocode-rules-specifications.version>
+        <!-- temporary version waiting for real automatic release in creedengo repository -->
+        <creedengo-rules-specifications.version>1.6.4</creedengo-rules-specifications.version>
 
     </properties>
 
     <dependencies>
         <dependency>
             <groupId>${project.groupId}</groupId>
-            <artifactId>ecocode-rules-specifications</artifactId>
-            <version>${ecocode-rules-specifications.version}</version>
+            <artifactId>creedengo-rules-specifications</artifactId>
```

Comment:
this renaming task has to be done in another PR once "ecocode" repository will be renamed (change groupId to keep "io.ecocode" in this PR, but restore it in your 2nd PR)

Reply:
i thinking the best thing is to use "io.greencodeinitiative" like used in package name inside source code